UnityProject/Assets/Scripts/Systems/Antagonists/AntagManager.cs

```diff
@@ -25,9 +25,9 @@ public class AntagManager : MonoBehaviour
 		/// <summary>
 		/// All active antagonists
 		/// </summary>
-		private List<SpawnedAntag> ActiveAntags = new List<SpawnedAntag>();
+		private List<SpawnedAntag> activeAntags = new List<SpawnedAntag>();
 
-		public List<SpawnedAntag> CurrentAntags => ActiveAntags;
+		public List<SpawnedAntag> CurrentAntags => activeAntags;
```

Comment:
call it ``ActiveAntags``, sorry I thought I was clear enough in my previous comment. It is better to keep the name and only differentiate them by the case style so you know instantly what it is and what access level it has.